### PR TITLE
@ashfurrow => beginners four

### PR DESCRIPTION
**Pull request by orta**
Making a Show View Controller in console.

### CODE REVIEW

File: Beginners/Lesson Four/Lesson Four.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+struct Artwork {
+    var name: String
+    var medium: String
+    var availability: String
+    var date: String
+    var height: Int
+
+    func printArtworkSubtitle(artwork: Artwork) {
```
Comment (ashfurrow) [inline Beginners/Lesson Four/Lesson Four.playground/Contents.swift]: Should this be both an instance method _and_ accept an `artwork` parameter?

Comment (orta) [inline Beginners/Lesson Four/Lesson Four.playground/Contents.swift]: nooope, might just outright remove

File: Beginners/Lesson Four/Lesson Four.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    func drawShowRange() {
+        let formatter = NSDateFormatter()
+        formatter.dateFormat = "MM-dd"
+        let end = formatter.stringFromDate(self.openingDate)
+        let start = formatter.stringFromDate(self.closingDate)
+        print("\(end) - \(start)")
+    }
+}
```
Comment (ashfurrow) [inline Beginners/Lesson Four/Lesson Four.playground/Contents.swift:42]: No `ausstellungsdauer`?

**Comment by ashfurrow** [comment]
Looks great! Feel free to merge when ready. 

Such text

File: Beginners/Lesson Four/Lesson Four.playground/Contents.swift (hunk 2 above)
Comment (orta) [inline Beginners/Lesson Four/Lesson Four.playground/Contents.swift:42]: I know :D

**Pull request merged**